.github/CODEOWNERS

```diff
@@ -4,78 +4,79 @@
+/plugins/catalog-backend-module-bitbucket-cloud   @backstage/reviewers @backstage/catalog-core @pjungermann
+/plugins/catalog-backend-module-msgraph           @backstage/reviewers @backstage/catalog-core @pjungermann
+/plugins/catalog-graph                            @backstage/reviewers @backstage/catalog-core @backstage/sda-se-reviewers
+/plugins/circleci                                 @backstage/reviewers @adamdmharvey
+/plugins/cloudbuild                               @backstage/reviewers @trivago/ebarrios
+/plugins/code-coverage                            @backstage/reviewers @alde @nissayeva
+/plugins/code-coverage-backend                    @backstage/reviewers @alde @nissayeva
+/plugins/cost-insights                            @backstage/reviewers @backstage/silver-lining
+/plugins/cost-insights-*                          @backstage/reviewers @backstage/silver-lining
+/plugins/events-backend                           @backstage/reviewers @pjungermann
+/plugins/events-backend-module-aws-sqs            @backstage/reviewers @pjungermann
+/plugins/events-backend-module-azure              @backstage/reviewers @pjungermann
+/plugins/events-backend-module-bitbucket-cloud    @backstage/reviewers @pjungermann
+/plugins/events-backend-module-gerrit             @backstage/reviewers @pjungermann
+/plugins/events-backend-module-github             @backstage/reviewers @pjungermann
+/plugins/events-backend-module-gitlab             @backstage/reviewers @pjungermann
+/plugins/events-backend-test-utils                @backstage/reviewers @pjungermann
+/plugins/events-node                              @backstage/reviewers @pjungermann
+/plugins/explore                                  @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/explore-react                            @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/fossa                                    @backstage/reviewers @backstage/sda-se-reviewers
+/plugins/gcalendar                                @backstage/reviewers @szubster @ptychu @kielosz @alexrybch
+/plugins/git-release-manager                      @backstage/reviewers @erikengervall
+/plugins/home                                     @backstage/reviewers @backstage/techdocs-core
+/plugins/ilert                                    @backstage/reviewers @yacut
+/plugins/jenkins                                  @backstage/reviewers @timja
+/plugins/jenkins-backend                          @backstage/reviewers @timja
+/plugins/kafka                                    @backstage/reviewers @nirga @andrewthauer
+/plugins/kafka-backend                            @backstage/reviewers @nirga @andrewthauer
+/plugins/kubernetes                               @backstage/reviewers @backstage/warpspeed
+/plugins/kubernetes-*                             @backstage/reviewers @backstage/warpspeed
+/plugins/microsoft-calendar                       @Abhay-soni-developer
```

Comment:
Without actually getting write access to the repo we've found that adding to this file for external contributors won't do anything so I'm not sure it make sense to make this change.

Reply:
Even though it doesn't work as intended it's still useful to us so we can see the owner easily :) We have also spoken about writing some tooling ourselves to ping the relevant code owners from a bot account in a comment for notifications rather than permissions.

Reply:
Ah, that makes sense to me 🚀